core/certs.go

```diff
@@ -68,9 +72,10 @@ func loadKongCerts(config *tomlConfig, url string, secretBaseURL string, c *http
 	return nil
 }
 
-func getCertKeyPair(config *tomlConfig, secretBaseURL string, c *http.Client) (string, string, error) {
+// ----------------------------------------------------------
+func getCertKeyPair(config *tomlConfig, secretBaseURL string, c *http.Client, debug bool) (string, string, error) {
 
-	t, err := getSecret(config.SecretService.TokenPath)
+	t, err := getSecret(config.SecretService.TokenFolderPath + "/" + config.SecretService.VaultInitParm)
```

Comment:
ideally you would use `filepath.Join` like in the example here: https://golang.org/pkg/path/filepath/#Join
This will work on all operating systems and handle edge cases correctly

Reply:
OK will fix that, I actually used filepath in my pkisetup code. This one is a legacy I did not write, so I did not check the function.

Reply:
OK done